src/expr/node_builder.h

```diff
@@ -774,9 +776,11 @@ template <unsigned nchild_thresh>
 void NodeBuilder<nchild_thresh>::realloc(size_t toSize) {
   AlwaysAssert(toSize > d_nvMaxChildren,
                "attempt to realloc() a NodeBuilder to a smaller/equal size!");
-  Assert( toSize < (1lu << CVC4__EXPR__NODE_VALUE__NBITS__NCHILDREN),
-          "attempt to realloc() a NodeBuilder to size %u (beyond hard limit of %u)",
-          toSize, (1lu << CVC4__EXPR__NODE_VALUE__NBITS__NCHILDREN) - 1 );
+  Assert(toSize < (((size_t)1u) << expr::NodeValue::NBITS_NCHILDREN),
```

Comment:
```suggestion
  Assert(toSize < (static_cast<size_t>(1u) << expr::NodeValue::NBITS_NCHILDREN),
```
[See our guidelines about C-style casts.](https://github.com/CVC4/CVC4/wiki/Code-Style-Guidelines#casting) The suggestion also removes the `u` from `1u` to be more consistent with the other definitions.